app/controllers/organizations_controller.rb

```diff
@@ -26,6 +28,9 @@ def create
   end
 
   def edit
+    @form_label = "col-12 col-md-4 col-lg-2"
+    @form_text_fields = "col-12 col-md-8 col-lg-10 mx-auto"
```

Comment:
I also have the same doubt about this... If they are going to be used in multiple places and we want to define them, wouldn't it be better to have them in a shared/global file?

Reply:
Also, one thing I noticed while reviewing, is that if there is an error in the form all the bootstrap classes for all the form disappear, while if you add those classes in the label/text_field (`<%= f.label :org_address, :class => "col-12 col-md-4 col-lg-2" %>
                    <%= f.text_field :org_address, :class => "col-12 col-md-8 col-lg-10 mx-auto" %>`) It is fine after the validation.
In my opinion I prefer to have the classes in the view, not defined in the controller. I find it more intuitive and easy to find/change/maintain. But of course I adapt to what all of you prefer

Reply:
Hey @saradotlog , same sentiment about this one. I have not worked on other resources and cannot presume that this grid layout will work best for them. We discussed building something simple in anticipation of it being edited and changed once we have a better picture of our product. I agree that some global style guides will be useful, let's discuss it more at the meeting

Reply:
Also, the video I shared was from when the code had the classes inside the label/text field. The warping still occurs before a variable pattern was added.

Reply:
So I fetched the new code and tested to edit the form again.
This is what I meant about the variables: 
When loading the first time "edit organisation":
![View1](https://user-images.githubusercontent.com/2697750/115253195-6fb99b80-a12c-11eb-94df-1577ad0349ba.jpg)
Everything looks correct with the bootstrap grid classes added.
After it, I try to update the form, leaving one field empty to force an error. The field-with-errors appears in the field I left empty, so the grid breaks. But also the grid classes that are supposed to be in the fields don't appear in the html:
![View2](https://user-images.githubusercontent.com/2697750/115253669-e2c31200-a12c-11eb-9d20-72ed62015a08.jpg)
Instead if I add the classes directly in the edit.html.erb file like this 
```
<div class="row organizations-field py-2">
        <%= f.label :org_name, :class => "col-12 col-md-4 col-lg-2" %>
        <%= f.text_field :org_name, :class => "col-12 col-md-8 col-lg-10 mx-auto" %>
</div>
```
and try to edit the form again, the only classes that seems to appear are the ones added in the view, not the ones in the controller:
![view3](https://user-images.githubusercontent.com/2697750/115254127-59600f80-a12d-11eb-881c-23704e8bd3ea.jpg)

Maybe the form validations doesn't go through the controller again and those variables are empty when reloading? 
Anyway, as I said, I'm open to everything, but I find much easier and intuitive to add the classes in the appropriate view.

Reply:
Oh oh now i get what you were saying yesterday @saradotlog

Reply:
Thanks for these screenshots @saradotlog ! Have moved the classes to the view file instead